### PR TITLE
Add messaging functionality

### PR DESCRIPTION
**Pull request by echanlee**

### CODE REVIEW

File: src/Messages.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    this.state = {
+        userId: this.props?.location?.state?.id,
+        messages : [],
+        messageSender: [],
+        timeStamps: [],
+        message :"",
+        room: this.props?.location?.state?.friendConvoId,
+        currentName: this.props?.location?.state?.currentName,
+        friendName: this.props?.location?.state?.friendName,
+        friendId:  this.props?.location?.state?.friendId,
```
Comment (jj7lin) [inline src/Messages.js:21]: just here to say good job, keep up the good work

**Review by jj7lin — COMMENTED** [review]
just here to say good job, keep up the good work

File: server/matches.py [hunk 2]
```diff
@@ -10,22 +10,47 @@ def matchUser(userId):
...
                 from ppFriends.Conversation c join ppFriends.Profile p \
                 on (c.userOne = p.userId or c.userTwo = p.userId) where userId != %s"
             cursor.execute(sql, (userId,))
-            conversationIds = []
             userIds = []
             firstnames = []
+            
             for pos in cursor.fetchall():
-                conversationIds.append(pos[0])
                 if pos[1] not in userIds:
```
Comment (wclui) [inline server/matches.py]: isn't pos[1] referring to first name? should this be pos[0]?

File: server/myapp.py [hunk 3]
```diff
@@ -61,7 +65,43 @@ def questionnaire():
...
+def getMessages():
+    if request.method == 'POST':
+        param = request.get_json('convoId')
+        return messages.getMessages(param['convoId'])   
+
+@app.route('/sendMessage', methods=['POST'])
+def sendMessage():
+    if request.method == 'POST':
+        param = request.get_json('convoId')
+        print(param)
```
Comment (wclui) [inline server/myapp.py]: do we wanna keep this print statement?

**Review by wclui — CHANGES_REQUESTED** [review]

**Pull request merged**